es6/README.md

```diff
@@ -252,7 +252,124 @@ For more on the spread operator, read [_Learning ES6: Rest & Spread Operators_](
 
 ## Objects
 
-Coming soon...
+When a variable name matches the name of the object key in an object literal, use [property value shorthand](http://www.eventbrite.com/engineering/learning-es6-enhanced-object-literals/#property-value-shorthand) (eslint: [`object-shorthand`](http://eslint.org/docs/rules/object-shorthand)):
+
+```js
+let name = 'Eventbrite';
+
+// good
+let data = {
+    name
+};
+
+// bad (duplicates key and variable name)
+let data = {
+    name: name
+};
+```
+
+Group any object literal property value shorthands at the beginning of the object literal so that it's easier to see which properties are using the shorthand:
+
+```js
+let name = 'Eventbrite';
+let location = 'San Francisco, CA';
+
+// good
+let data = {
+    name,
+    location,
+    ceo: 'Julia Hartz',
+    founders: ['Julia Hartz', 'Kevin Hartz', 'Renaud Visage']
+};
+
+// bad (shorthands aren't at the top)
+let data = {
+    name,
+    ceo: 'Julia Hartz',
+    founders: ['Julia Hartz', 'Kevin Hartz', 'Renaud Visage'],
+    location
+};
+```
+
+When creating object literals with dynamic property names, use [computed property keys](http://www.eventbrite.com/engineering/learning-es6-enhanced-object-literals/#computed-property-keys) (eslint: [`object-shorthand`](http://eslint.org/docs/rules/object-shorthand)):
+
+```js
+let name = 'Eventbrite';
+let location = 'San Francisco, CA';
+let leaderName = 'ceo';
+
+// good
+let data = {
+    name,
+    location,
+    [leaderName]: 'Julia Hartz',
+    founders: ['Julia Hartz', 'Kevin Hartz', 'Renaud Visage']
+};
+
+// bad (doesn't leverage computed property keys)
+let data = {
+    name,
+    location,
+    founders: ['Julia Hartz', 'Kevin Hartz', 'Renaud Visage']
+};
+
+data[leaderName] = 'Julia Hartz';
+```
+
+When defining methods on an object literal, use [method definition shorthand](http://www.eventbrite.com/engineering/learning-es6-enhanced-object-literals/#method-definition-shorthand) (eslint: [`object-shorthand`](http://eslint.org/docs/rules/object-shorthand)):
+
+```js
+let name = 'Eventbrite';
+let location = 'San Francisco, CA';
+let leaderName = 'ceo';
+
+// good
+let data = {
+    name,
+    location,
+    [leaderName]: 'Julia Hartz',
+    founders: ['Julia Hartz', 'Kevin Hartz', 'Renaud Visage'],
+    getDisplay() {
```

Comment:
Naw because of how arrow functions work with `this` it wouldn't make sense. That's why method shorthand syntax is preferred.